x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/components/time_range_step/time_range.tsx

```diff
@@ -82,27 +80,27 @@ export const TimeRangeStep: FC<StepProps> = ({ setCurrentStep, isCurrentStep })
     });
   }
 
-  useEffect(() => {
-    if (mlJobService.currentJob !== undefined) {
-      (async (index: string, timeFieldName: string | undefined, query: object) => {
-        const resp = await ml.getTimeFieldRange({
-          index,
-          timeFieldName,
-          query,
-        });
-        setTimeRange({
-          start: resp.start.epoch,
-          end: resp.end.epoch,
-        });
-        // wipe the cloning job
-        mlJobService.currentJob = undefined;
-      })(
-        kibanaContext.currentIndexPattern.title,
-        kibanaContext.currentIndexPattern.timeFieldName,
-        kibanaContext.combinedQuery
-      );
-    }
-  }, []);
+  // useEffect(() => {
```

Comment:
Can you add a comment why this is commented out and why/when it will be re-enabled?

Reply:
ah no, left in by mistake.